tracer/src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnosticObserver.cs

```diff
@@ -557,7 +557,7 @@ private void OnRoutingEndpointMatched(object arg)
                 {
                     span.ResourceName = resourceName;
                     tags.AspNetCoreRoute = normalizedRoute;
-                    span.SetTag(Tags.HttpRoute, normalizedRoute);
+                    tags.HttpRoute = normalizedRoute;
```

Comment:
Good catch. It you do a search for `Tags.HttpRoute` you'll find we're setting this in several other places that we can probably do the same thing:

- Further up (line 403)
- In all the ASP.NET (non-core) integrations

Reply:
done in [b34ef11](https://github.com/DataDog/dd-trace-dotnet/pull/3907/commits/b34ef1119e7a1f1cf147d28f696c2af9cdbea395)